mantle/kola/tests/ignition/luks.go

```diff
@@ -176,6 +187,68 @@ func runTest(c cluster.TestCluster, tpm2 bool, threshold int, killTangAfterFirst
 	ut.LUKSSanityTest(c, tangd, m, tpm2, killTangAfterFirstBoot, rootPart)
 }
 
+func runCexTest(c cluster.TestCluster) {
+	var err error
+	var m platform.Machine
+
+	// To prevent the test to fail the whole run on s390x machine that does not have Cex Device
+	cex_uuid := os.Getenv("KOLA_CEX_UUID")
+	if cex_uuid == "" {
+		c.Skip("No CEX device found in KOLA_CEX_UUID env var")
+	}
+
+	ignition := conf.Ignition(`{
+		"ignition": {
+			"version": "3.5.0"
+		},
+		"kernelArguments": {
+			"shouldExist": [
+				"rd.luks.key=/etc/luks/cex.key"
+		]
+		},
+		"storage": {
+			"luks": [
+				{
+					"name": "root",
+					"device": "/dev/disk/by-label/root",
+					"cex": {
+						"enabled": true
+					},
+					"label": "root",
+					"wipeVolume": true
+				}
+			],
+			"filesystems": [
+				{
+					"device": "/dev/mapper/root",
+					"format": "xfs",
+					"wipeFilesystem": true,
+					"label": "root"
+				}
+			]
+		}
+	}`)
+
+	opts := platform.QemuMachineOptions{
+		Cex: true,
+	}
+	opts.MinMemory = 8192
+
+	switch pc := c.Cluster.(type) {
+	case *qemu.Cluster:
+		m, err = pc.NewMachineWithQemuOptions(ignition, opts)
+	default:
+		panic("Unsupported cluster type")
+	}
+
```

Comment:
Here, add:

```
	// copy over kolet into the machine
	if err := kola.ScpKolet([]platform.Machine{m}); err != nil {
		c.Fatal(err)
	}
	LocalTests(c)
```

This will run the growfs test code.